### PR TITLE
[lectl] Add target for installing new package

### PR DESCRIPTION
**Pull request by vadim-hleif**
## what
* add targets which can install `lectl`

## why
* Make it easier to triage Lets Encrypt certificates

## references
- https://github.com/cloudposse/packages/issues/33

### CODE REVIEW

File: install/Makefile [hunk 1]
```diff
@@ -141,6 +142,13 @@ export HUGO_VERSION ?= 0.45.1
 hugo: hugo-$(OS)
 	@exit 0
 
+export LECTL_VERSION ?= 0.16
+# Releases: https://github.com/sahsanu/lectl/releases
+## Install lectl script to check issued certificates by Let's Encrypt on CTL
+lectl:
+	$(CURL) -o - https://github.com/sahsanu/lectl/archive/v$(LECTL_VERSION).tar.gz | tar -zxO  lectl-${LECTL_VERSION}/lectl > $(INSTALL_PATH)/lectl
```
Comment (osterman) [inline install/Makefile]: No need to download the archive. Just use the `raw` url.

File: install/Makefile [hunk 2]
```diff
@@ -141,6 +142,12 @@ export HUGO_VERSION ?= 0.45.1
 hugo: hugo-$(OS)
 	@exit 0
 
+# Releases: https://github.com/sahsanu/lectl/releases
+## Install lectl script to check issued certificates by Let's Encrypt on CTL
+lectl:
+	$(CURL) -o - https://raw.githubusercontent.com/sahsanu/lectl/master/lectl > $(INSTALL_PATH)/lectl
```
Comment (osterman) [inline install/Makefile]: Parameterize version.

File: install/Makefile [hunk 3]
```diff
@@ -142,6 +143,13 @@ export HUGO_VERSION ?= 0.45.1
 hugo: hugo-$(OS)
 	@exit 0
 
+export LECTL_VERSION ?= 0.16
+# Releases: https://github.com/sahsanu/lectl/releases
+## Install lectl script to check issued certificates by Let's Encrypt on CTL
+lectl:
```
Comment (osterman) [inline install/Makefile]: Add -sSL arguments so we properly handle redirects and failures

Comment (vadim-hleif) [reply install/Makefile]: `$(CURL)` it's `CURL ?= curl --fail -sSL`

Comment (osterman) [reply install/Makefile]: Oops. You're right =)

**Pull request merged**